### PR TITLE
Feat: navbar responsive

### PR DESCRIPTION
**Pull request by Amal95Salah**

### CODE REVIEW

File: src/components/NavBar/Navbar.js [hunk 1]
```diff
@@ -72,6 +75,43 @@ function Navbar() {
               )}
             </ul>
           </div>
+          <div className="md:hidden">
+            <Dropdown label="" inline>
```
[the remark this answers is not in the document]
Comment (Amal95Salah) [reply src/components/NavBar/Navbar.js:79]: label here means lable for button, do we need button?
I think the arrow is good enough 
button will destroy the style

File: src/components/NavBar/Navbar.js [hunk 2]
```diff
@@ -72,6 +75,43 @@ function Navbar() {
               )}
             </ul>
           </div>
+          <div className="md:hidden">
+            <Dropdown label="" inline>
+              {currentUser ? (
+                <Dropdown.Header onClick={() => navigate('/profile')}>
```
Comment (RageOfKurd) [inline src/components/NavBar/Navbar.js:81]: could you add ( 'cursor-pointer') class so when hovered the cursor visibility changes.

File: src/components/NavBar/Navbar.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (Amal95Salah) [reply src/components/NavBar/Navbar.js]: Resolved

File: src/components/NavBar/Navbar.js (hunk 2 above)
Comment (RageOfKurd) [reply src/components/NavBar/Navbar.js:81]: ur right I didn't remember that XD

**Review by RageOfKurd — APPROVED** [review]
awesome looks great

**Pull request merged**